Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -97,24 +103,40 @@ bool ScrollAnchoringController::didFindPriorityCandidate(Document& document)
     return false;
 }
 
+static bool elementHasScrollAnchoringController(Element& element)
```

Comment:
We want to avoid descending into elements handled by ScrollAnchoringControllers, so we want to include overflow:hidden elements in that. Unless you meant there is a more concise way of checking if an element has a scrollable area than this?

Reply:
Changed this to check `hasPotentiallyScrollableOverflow()`.